csharp-intermediate-topics/TupleAliasInCSharp/TupleAliasInCSharp/TupleAlias.cs

```diff
@@ -0,0 +1,35 @@
+﻿namespace TupleAliasInCSharp
+{
+    using tupleAliasUsing = (int num1, string str, double num2);
```

Comment:
The tuple alias should be UpperCamelCase.

Reply:
@karimkod - This PR has been updated to reflect this change.